Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -583,9 +585,9 @@ internal string ExtractMatches(string source, string outputSeparator = null)
                 }
             }
 
-            private static string ExtractTextFromMatch(Match match)
+            private string ExtractTextFromMatch(Match match)
             {
-                var errorMessageGroup = match.Groups[ErrorMessageGroupName];
+                Group errorMessageGroup = match.Groups[GroupName];

```

Comment:
>errorMessageGroup [](start = 22, length = 17)

The variable `errorMessageGroup` may be misnomer here because we no longer use this method for matching against `ErrorMessageGroupName`.